cmp/grid/index.js

```diff
@@ -8,3 +8,9 @@
 export * from './Grid';
 export * from './GridModel';
 export * from './GridStateModel';
+
+export * from './columns/Core';
+export * from './columns/DatesTimes';
+export * from './columns/ExportFormat';
+export * from './columns/Column';
+export * from './columns/ColumnGroup';
```

Comment:
I would vote for leaving `columns/index.js` in place and then re-exporting that from here. Would help with app-level updates - in that the `/cmp/grid/columns -> `/cmp/grid` import change would still be a good idea but be optional. More so though it seems like helpful to have sub-packages like this one with several nested files manage their own "directory listing".  Thoughts? Would defer to you and @lbwexler if you've discussed and don't want to do this.